src/cgr_gwas_qc/workflow/modules/bcf.smk

```diff
@@ -0,0 +1,107 @@
+from cgr_gwas_qc import load_config
+from math import ceil
+
+cfg = load_config()
+
+############################
+# General BCF import rules
+###############################
+
+
+def _create_unknown_sex(wildcards):
+    path_to_unknown_sex_lst = temp("unknown_sex.lst")
```

Comment:
The function is used in `params` section of rule `convert_bcf_to_plink_bed` . This rule is reused with BCF entry point `cfg.config.user_files.bcf` as well. Directing the output to `sample_level` directory in case of GTC input with `cfg.config.workflow_params.convert_gtc2bcf ` should work fine but with `cfg.config.user_files.bcf`  causes an error since this might be the first step and  `sample_level`  directory won't exist. 

There may be still a different implementation possible to allow redirecting to  `sample_level`. Since it is writing as `temp` file, I believe not a big issue?

Reply:
With f72247e7885dd206648362d55f6a0de874c068cc, it will be written to tempdir and also removed at the end. Thanks.